gremlin-go/driver/logger.go

```diff
@@ -96,6 +96,12 @@ func (logHandler *logHandler) logf(verbosity LogVerbosity, errorKey errorKey, v
 	}
 }
 
+func (logHandler *logHandler) logErr(verbosity LogVerbosity, err error) {
+	if verbosity >= logHandler.verbosity {
+		logHandler.logger.Log(verbosity, err.Error())
```

Comment:
I don't think this method is required.

The issue is that we were invoking calls such as `log.logger.Logf(Info, "OK %v===>%v", response.responseStatus, data)`

The API requires `func (logHandler *logHandler) logf(verbosity LogVerbosity, errorKey errorKey, v ...interface{})` where we need a parameter of type `errorKey`, which is a enum specified elsewhere in the project. The issue is that `errorKey` is technically a type alias for a string, so there is no strict type enforcement, allowing you to pass in `OK %v===>%v`, when this will cause logging to fail.

To fix this all we need is a new errorKey that maps to the string value of `OK %v===>%v`.

Reply:
`log.logger.Logf` ignore LogVerbosity so should not be used. 

Translation happening in https://github.com/lyndonbauto/tinkerpop/blob/gremlin-go-ms5/gremlin-go/driver/logger.go#L94-L95, then passed to `logHandler.logger.Logf` so `logErr` should work